pyspectral/rayleigh.py

```diff
@@ -228,16 +230,29 @@ def get_reflectance_lut(self):
             self._azid_coord = lut_vars[2]
             self._satz_sec_coord = lut_vars[3]
             self._sunz_sec_coord = lut_vars[4]
-        return self._rayl, self._wvl_coord, self._azid_coord,\
-            self._satz_sec_coord, self._sunz_sec_coord
+        return self._rayl, self._wvl_coord, self._azid_coord, \
+               self._satz_sec_coord, self._sunz_sec_coord
```

Comment:
Please leave as it was. Is it some kind of auto correct you have?

Reply:
```suggestion
        return (self._rayl, self._wvl_coord, self._azid_coord,
                self._satz_sec_coord, self._sunz_sec_coord)
```

Reply:
Yes, thanks, choose that @simonrp84